lib/git/lib.rb

```diff
@@ -396,6 +396,7 @@ def diff_index(treeish)
     end
 
     def ls_files(location=nil)
+      location ||= '.'
```

Comment:
Why not do this in the method arguments?

```ruby
def ls_files(location='.')
...
end
```

Reply:
I thought about doing this, but then on [ls_remote](https://github.com/ruby-git/ruby-git/pull/350/files/965ffeea3aba0adb69627157d1ea81e7775ac004#diff-6269137dee54cb0723d82224f3c5223cL410) it was handled this way, so I kept the standard of code that was already there.